meteor/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:experimental
 # BUILD IMAGE
-FROM node:14
+FROM meteor/node:14.21.4
 RUN curl "https://install.meteor.com/?release=2.12" | sh
```

Comment:
```suggestion
RUN curl "https://install.meteor.com/?release=2.13.3" | sh
```

Reply:
There is another line like this in `.github/actions/` somewhere, which should be updated too

Reply:
Thanks, found it